test/ui/ApplicantTest.ts

```diff
@@ -20,4 +20,27 @@ describe('Applicant UI Test', async () => {
 
         await driver.quit();
     });
+
+    it('As an applicant, I am able to click on a job role and see the details', async () => {
+        const driver = new Builder().
+        withCapabilities(Capabilities.chrome()).
+        build();
+
+        const endPoint: string = 'job-roles';
+        await driver.get(baseUrl + endPoint);
+        
+        const element = driver.findElement(By.id('roleNameInList'));
+        driver.executeScript("arguments[0].scrollIntoView(true);", element);
+
+        await driver.findElement(By.linkText("Software Engineer")).click();
+
+        const endPoint2: string = 'job-roles/1';
+        await driver.get(baseUrl + endPoint2);
+        
+       const name = await driver.findElement(By.id('roleName')).getText();
+        expect(name).to.equal('Software Engineer');
+
+        await driver.quit();
+
+    });
```

Comment:
Maybe a test to check if the link to the sharepoint pdf takes the user to the correct place?

Theres a function in selenium where you can do: **driver. getCurrentUrl();** to grab the current URL from the browser

Reply:
To add on above: if you're not a logged in user, it will take you to the Kainos login screen, so maybe just an assertion that verifies you get to the login page

Reply:
just wrote the test case. Let me know if you think the test I wrote properly addresses this!

Reply:
Looks good :)